spec/controllers/v2/users/account_controller_spec.rb

```diff
@@ -86,7 +92,7 @@
         end
 
         it 'sends a confirmation email and two notification emails' do
-          expect(UsersMailer.deliveries.size).to eq(3)
```

Comment:
Please don't delete this check. I'd still like to know if all emails have actually been delivered.